pennylane/ops/qubit/qchem_ops.py

```diff
@@ -464,3 +464,160 @@ def _matrix(cls, *params):
     def adjoint(self):
         (theta,) = self.parameters
         return DoubleExcitationMinus(-theta, wires=self.wires)
+
+
+class OrbitalRotation(Operation):
+    r"""OrbitalRotation(varphi, wires)
```

Comment:
Could not be just `phi`?

Reply:
Yes, it can be. I have now have changed the  `phi` in both the codebase and relevant images.